projects/go-lib/src/lib/components/go-icon-button/go-icon-button.component.scss

```diff
@@ -0,0 +1,18 @@
+@import '~@tangoe/gosheets/base/variables';
+@import '~@tangoe/gosheets/base/mixins';
+
+.go-icon-button {
```

Comment:
We should probably throw a `background` attribute on here, just to enforce where this is used.
`background: $theme-light-bg;`